client/components/plans/plan/index.jsx

```diff
@@ -40,8 +40,14 @@ module.exports = React.createClass( {
 		}
 	},
 
+	getComparePlansUrl: function() {
+		const { site } = this.props;
+		var siteSuffix = site ? site.slug : '';
```

Comment:
Super minor, but we're trying to avoid ES6 syntax (like const or object destructuring) in files that haven't been fully rewritten in ES6 (there is another erroneous instance of this on line 51). This should be:

```js
var site = this.props.site,
  siteSuffix = site ? site.slug : '';
```